nss-nspr/Patches/ldflags.diff

```diff
@@ -1,17 +1,26 @@
-From 418135dcb6e1a9fc7e1b4b59a32f42959a9b7123 Mon Sep 17 00:00:00 2001
-From: John Levon <john.levon@joyent.com>
-Date: Thu, 15 Nov 2018 22:09:47 +0000
-Subject: [PATCH] Apply $LDFLAGS when building shared libraries
-
----
- nss/coreconf/SunOS5.mk | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/nss/coreconf/SunOS5.mk b/nss/coreconf/SunOS5.mk
-index ce9e2cb..84e2093 100644
---- a/nss/coreconf/SunOS5.mk
-+++ b/nss/coreconf/SunOS5.mk
-@@ -79,7 +79,7 @@ endif
+diff -ru a/nss/cmd/shlibsign/sign.sh b/nss/cmd/shlibsign/sign.sh
+--- a/nss/cmd/shlibsign/sign.sh	Mon Jun 20 13:11:28 2016
++++ b/nss/cmd/shlibsign/sign.sh	Thu Jan 30 18:39:24 2025
+@@ -37,6 +37,15 @@
+     export LIBPATH
+     SHLIB_PATH=${1}/lib:${4}:$SHLIB_PATH
+     export SHLIB_PATH
++    # XXX SmartOS - Add non-strap libraries if we aren't building strap.
++    if [[ "$STRAP" == "" ]]; then
++	if [[ "$USE_64" == "1" ]]; then
++            LD_LIBRARY_PATH=${DESTDIR}/lib/64:${DESTDIR}/usr/lib/64:$LD_LIBRARY_PATH
```

Comment:
I have used LD_LIBRARY_PATH_64 in such cases, so the 32-bit world is not getting unexpected errors

Reply:
Problem is, the script later down requires further additions to LD_LIBRARY_PATH, so I did it this way to minimize diffs.  Esp. important if/when we update this again.

Reply:
ah, I see. ok, than it makes sense.